Sources/MongoSwift/ReadConcern.swift

```diff
@@ -2,53 +2,55 @@ import CLibMongoC
 
 /// A struct to represent a MongoDB read concern.
 public struct ReadConcern: Codable {
+    /// Local ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-local/.
+    public static let local = ReadConcern(.local)
+    /// Available ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-available/.
+    public static let available = ReadConcern(.available)
+    /// Linearizable ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-majority/.
+    public static let linearizable = ReadConcern(.linearizable)
+    /// Majority ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-linearizable/.
+    public static let majority = ReadConcern(.majority)
+    /// Snapshot ReadConcern, see https://docs.mongodb.com/master/reference/read-concern-snapshot/.
+    public static let snapshot = ReadConcern(.snapshot)
+    /// Empty ReadConcern.
+    public static let empty = ReadConcern(nil)
+
+    public static func other(_ level: Level) -> ReadConcern {
```

Comment:
rather than having static `other` methods, I think using `init` like we have for `Level` serves us fine.

Reply:
I think the motivation for having an `other` method is so you can inline syntax like this:
`RunCommandOptions(readConcern: .other("blah"))`
instead of
`RunCommandOptions(readConcern: ReadConcern("blah"))`
Are we okay with the constructor/init syntax in this case?

Reply:
I personally find the second one (using the initializer) more straightforward, though I'm curious to see what everyone thinks. 

cc @kmahar @mbroadst

Reply:
i think i prefer the `other` method to emphasize this is only for use in an edge case and to push people toward using the static properties whenever possible.

Reply:
sgtm :+1: